site/en/docs/extensions/mv3/screen_capture/index.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Audio recording and screen capture"
+seoTitle: "Chrome Extensions: Audio recording and screen capture"
+date: 2023-04-14
+updated: 2023-04-14
+description: How to record audio or video from a tab, window or screen
+---
+
+This guide explains different approaches for recording audio and video from a tab, window or
+screen using APIs such as [chrome.tabCapture][tabcapture] or [getDisplayMedia][get-display-media].
+
+## Common Use Cases
+
+### Screen Recording {: #screen-recording}
+
+For screen recording, use the [getDisplayMedia][get-display-media] API. This provides the user with
```

Comment:
This and a later edit go together. The goal behind them is to speed comprehension by clarifying relationships in your content.

A writing best practice is to use specific rather than general language. If I've never heard of `getDisplayMedia()`, I may need to look it up to understand a sentence like this. Calling it specifically what is makes the sentence clearer. Writing 'call' instead of 'use' tells me specifically that `getDisplayMedia()` is a function or method. If I didn't have the verb 'call' I would use the words "`getDisplayMedia()` function" to say specifically what it is.

>, which triggers the dialog box shown below

This speeds up comprehension by relating the items in your work to each other.